packages/react-sdk/README.md

```diff
@@ -325,6 +366,61 @@ const WrappedSignupButton = withOptimizely(SignupButton)
+* `user: User` The current user associated with this client instance
+* `setUser(userInfo: User): void` Call this to update the current user
+* `onUserUpdate(handler: (userInfo: User) => void): () => void` Subscribe a callback to be called when this instance's current user changes. Returns a function that will unsubscribe the callback.
+* `activate(experimentKey: string, overrideUserId?: string, overrideAttributes?: UserAttributes): string | null` Activate an experiment, and return the variation for the given user.
+* `getVariation(experimentKey: string, overrideUserId?: string, overrideAttributes?: UserAttributes): string | null` Return the variation for the given experiment and user.
+* `getFeatureVariables(featureKey: string, overrideUserId?: string, overrideAttributes?: UserAttributes): VariableValuesObject`: Decide and return variable values for the given feature and user
+* `getFeatureVariableString(featureKey: string, variableKey: string, overrideUserId?: string, overrideAttributes?: optimizely.UserAttributes): string | null`: Decide and return the variable value for the given feature, variable, and user
+* `getFeatureVariableInteger(featureKey: string, variableKey: string, overrideUserId?: string, overrideAttributes?: UserAttributes): number | null` Decide and return the variable value for the given feature, variable, and user
+* `getFeatureVariableBoolean(featureKey: string, variableKey: string, overrideUserId?: string, overrideAttributes?: UserAttributes): boolean | null` Decide and return the variable value for the given feature, variable, and user
+* `getFeatureVariableDouble(featureKey: string, variableKey: string, overrideUserId?: string, overrideAttributes?: UserAttributes): number | null` Decide and return the variable value for the given feature, variable, and user
+* `isFeatureEnabled(featureKey: string, overrideUserId?: string, overrideAttributes?: UserAttributes): boolean` Return the enabled status for the given feature and user
+* `getEnabledFeatures(overrideUserId?: string, overrideAttributes?: UserAttributes): Array<string>`: Return the keys of all features enabled for the given user
+* `track(eventKey: string, overrideUserId?: string | EventTags, overrideAttributes?: UserAttributes, eventTags?: EventTags): void` Track an event to the Optimizely results backend
+* `setForcedVariation(experiment: string, overrideUserIdOrVariationKey: string, variationKey?: string | null): boolean` Set a forced variation for the given experiment, variation, and user
+* `getForcedVariation(experiment: string, overrideUserId?: string): string | null` Get the forced faration for the given experiment, variation, and user
+
+## Rollout or experiment a feature user-by-user
+
+To rollout or experiment on a feature by user rather than by random percentage, setup an attribute in Optimizely and create an Audience, which uses that attribute. Then pass along this attribute to the Provider.
```

Comment:
Is this talking about whitelisting? If so this is sort of incorrect.

Reply:
Let's say you have a feature that you want to run a beta on (user by user) or (customer by customer). My intention with putting this in the Readme was that many React Rollouts users were getting confused about how to leverage audiences and attributes and some weren't even sure if it was possible to rollout a feature user by user. Several users of the React SDK thought that userIds + Audiences was the right answer, but this is not the right answer.

This section was to try and resolve confusion around this topic by putting it front and center in the "User Case" section of the React docs. I also think this use case is the first thing that people will do after trying to toggle a feature on or off, well before they get to experimentation or random rollouts.

I spoke with Matt offline and we decided to just link to the rollouts docs. I do think though that because we haven't yet intermingled React into our official docs that this Readme should act as both a guide and an API reference.